server/src/github.ts

```diff
@@ -20,8 +21,11 @@ const schema = require('../github-schema.json');
 
 export class GitHub {
   private apollo: ApolloClient<NormalizedCacheObject>;
+  private jsonUrl: string;
 
-  constructor(uri = 'https://api.github.com/graphql') {
+  constructor(
+      uri = 'https://api.github.com/graphql',
```

Comment:
Maybe call this `gqlUrl` to distinguish it from the new `jsonUrl`.

Reply:
Done.